### PR TITLE
Add current vex game, turning point

### PR DESCRIPTION
**Pull request by m888r**

### CODE REVIEW

**Comment by PeterJohnson** [comment]
Did you generate the png file yourself?  What's the copyright on it?

File: src/main/java/edu/wpi/first/pathweaver/Game.java [hunk 1]
```diff
@@ -1,7 +1,9 @@
 package edu.wpi.first.pathweaver;
 
 public enum Game {
-  POWER_UP_2018("FIRST Power Up");
+  POWER_UP_2018("FIRST Power Up"),
+  TURNING_POINT_2018_19("VEX Turning Point"); // what am i doing wrong here
```
Comment (Octogonapus) [inline src/main/java/edu/wpi/first/pathweaver/Game.java]: Please remove this comment

File: src/main/java/edu/wpi/first/pathweaver/FxUtils.java [hunk 2]
```diff
@@ -133,7 +133,7 @@ public static void loadMainScreen(Scene scene, Class aClass) {
    * @return TextFormatter with only Numeric text.
    */
   public static TextFormatter<Object> onlyPositiveDoubleText() {
-    return textRestriction("^\\d*\\.?\\d*$");
+    return textRestriction("^\\d+\\.?\\d*$");
```
Comment (PeterJohnson) [inline src/main/java/edu/wpi/first/pathweaver/FxUtils.java]: This change is unrelated, please remove.

**Comment by M888R** [comment]
Image rendered by Jason from VEX community, there's no copyright.
[Forum Post](https://www.vexforum.com/index.php/33583-jason-s-turning-point-field-renders/0)

**Comment by Octogonapus** [comment]
The copyright is Jason's, but he permits use if you credit him with the image.

**Comment by AustinShalit** [comment]
Just an additional note for anyone who sees this in the future (https://xkcd.com/979/):  Per [our contributing guidelines](https://github.com/wpilibsuite/allwpilib/blob/master/CONTRIBUTING.md#licensing), we require all contributions to be `[relicensed] under the license for the WPILib project`.
